data-collections-exercises.md

```diff
@@ -3,95 +3,96 @@
 Starting with the following array...
 
 ```rb
-planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
+#planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 ```
 
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+#  planeteers[1]
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+#  planeteers.collect {|p| p + " Heart" }
```

Comment:
what a way of solving this problem !! 👏 , in this way you will add `Heart` for every element in the array so the array will be something like this 

```
["Earth Heart", "Wind Heart", "Captain Planet Heart", "Fire Heart", "Water Heart"] 
```

but the goal is to add another element at the end of the array, you can use the `push` method to achieve that 
`planeteers.push("Heart")`